nodejs-select2/nodejs-select2.spec

```diff
@@ -0,0 +1,67 @@
+# TODOs:
+#   - list doc files properly (%doc %{nodejs_sitelib}/%{npm_name}/README.md)
+#   - exclude unnecessary files from %files section
+
+%global npm_name select2
+
+Name: nodejs-%{npm_name}
+Version: 3.5.2
+Release: 1%{?dist}
+Summary: Browserify-ed version of Select2.
+License: unknown
```

Comment:
3.5.x appears to be dual-licensed as Apache 2 or GPLv2 (https://github.com/select2/select2/blob/stable/3.5/LICENSE), so `ASL 2.0 or GPLv2`